guides/asynchronous-tasks/readme.md

```diff
@@ -52,32 +52,32 @@ At any point in your program, you can start the reactor and a root task using th
 ```ruby
 Async do
 	3.times do |i|
-		sleep 1
+		sleep 1 + i/10.0
 		puts "Hello World #{i}"
 	end
 end
 ```
 
-This program prints "Hello World" 3 times. Before printing, it sleeps for 1 second. The total execution time is 3 seconds because the program executes sequentially.
+This program prints "Hello World" 3 times. Before printing, it sleeps for 1, then 1.1, then 1.2 seconds. The total execution time is 3.3 seconds because the program executes sequentially.
```

Comment:
Do you mind explaining the motivation to change this to `+ i/10.0`?

Reply:
Yes, I was trying to make the example more clear about when the stop happens. If all of the timers are firing at the 1 second mark, the timing of the stop is harder to follow.

But the tenths of a second are a bit cluttered for sure. I'd be happy to make those whole seconds or use some other approach for each thread to have a different timing value. Or I can put it back as it was if you'd prefer.

Reply:
I see, I think maybe just `sleep(i)` can work and is a bit easier on the eye?